climada/hazard/trop_cyclone.py

```diff
@@ -102,26 +102,37 @@ def __init__(self, pool=None):
 
     def set_from_tracks(self, tracks, centroids=None, description='',
                         model='H08', ignore_distance_to_coast=False,
-                        store_windfields=False):
+                        store_windfields=False, metric="equirect"):
         """Clear and fill with windfields from specified tracks.
 
-        Parameters:
-            tracks (TCTracks): tracks of events
-            centroids (Centroids, optional): Centroids where to model TC.
-                Default: global centroids.
-            description (str, optional): description of the events
-            model (str, optional): model to compute gust. Default Holland2008.
-            ignore_distance_to_coast (boolean, optional): if True, centroids
-                far from coast are not ignored. Default False
-            store_windfields (boolean, optional): If True, the Hazard object
-                gets a list `windfields` of sparse matrices. For each track,
-                the full velocity vectors at each centroid and track position
-                are stored in a sparse matrix of shape
-                (npositions,  ncentroids * 2), that can be reshaped to a full
-                ndarray of shape (npositions, ncentroids, 2). Default: False.
-
-        Raises:
-            ValueError
+        Parameters
+        ----------
+        tracks : TCTracks
+            Tracks of storm events.
+        centroids : Centroids, optional
+            Centroids where to model TC. Default: global centroids at 360 arc-seconds resolution.
+        description : str, optional
+            Description of the event set. Default: "".
+        model : str, optional
+            Model to compute gust. Currently only 'H08' is supported for the one implemented in
+            `_stat_holland` according to Greg Holland. Default: "H08".
+        ignore_distance_to_coast : boolean, optional
+            If True, centroids far from coast are not ignored. Default: False.
+        store_windfields : boolean, optional
+            If True, the Hazard object gets a list `windfields` of sparse matrices. For each track,
+            the full velocity vectors at each centroid and track position are stored in a sparse
+            matrix of shape (npositions,  ncentroids * 2) that can be reshaped to a full ndarray
+            of shape (npositions, ncentroids, 2). Default: False.
+        metric : str, optional
+            Specify an approximation method to use for earth distances:
+            * "equirect": Distance according to sinusoidal projection. Fast, but inaccurate for
```

Comment:
This is not relevant, since both methods are able to deal with tracks crossing the antimeridian. This was an issue with the legacy implementations in CLIMADA.